MultipleMatchError
==================

.. currentmodule:: sunpy.util.datatype_factory_base

.. autoexception:: MultipleMatchError
